flow360/component/simulation/models/material.py

```diff
@@ -171,6 +171,8 @@ def get_pressure(
         PressureType.Positive
             The calculated pressure.
         """
+        if temperature.units is u.degC or temperature.units is u.degF:
+            temperature = temperature.to("K")
```

Comment:
I think we can do `.to("K") always. Any temperature unit can be converted to "K".
```suggestion
        temperature = temperature.to("K")
```